packages/ckeditor5-link/tests/autolink.js

```diff
@@ -414,6 +425,27 @@ describe( 'AutoLink', () => {
 				'<paragraph>https://www.cksource.com []</paragraph>'
 			);
 		} );
+
+		it( 'should not undo auto-linking by pressing backspace after any other change has been made', () => {
```

Comment:
Could you add a link to the issue in the comment above this line? This test will seem very weird to someone looking at it in a year :).